Sources/Parsers/StoryboardParser.swift

```diff
@@ -28,90 +28,70 @@ public final class StoryboardParser {
     let customModule: String?
   }
 
+  enum XMLScene {
+    static let path = "/document/scenes/scene/objects/*[@sceneMemberID=\"viewController\"]"
```

Comment:
You're right that it's new logic, before we were just taking the first item in the `objects` array. But that actually lead to issue #17, where sometimes `viewController` isn't the first item in the array. Instead we now search for the actual viewcontroller in that logic, thus the `@sceneMemberID`.